ee/dtr/admin/upgrade.md

```diff
@@ -34,72 +27,74 @@ guarantees while upgrading between versions. While downgrades are not supported,
 
 Once you have the latest image on your machine (and the images on the target
-nodes if upgrading offline), run the upgrade command:
+nodes, if upgrading offline), run the upgrade command.
+
+> On which node should the upgrade command be run?
+>
+> The upgrade command can be run from any available node, as UCP is aware of of which worker nodes have replicas.
 
 ```bash
 docker run -it --rm \
   {{ page.dtr_org }}/{{ page.dtr_repo }}:{{ page.dtr_version }} upgrade
 ```
 
-By default the upgrade command runs in interactive mode and prompts you for
+By default the upgrade command runs in interactive mode and prompts for
 any necessary information. You can also check the
 [upgrade reference page](/reference/dtr/2.7/cli/upgrade/) for other existing flags.
 If you are performing the upgrade on an existing replica, pass the `--existing-replica-id` flag.
 
 The upgrade command will start replacing every container in your DTR cluster,
 one replica at a time. It will also perform certain data migrations. If anything
-fails or the upgrade is interrupted for any reason, you can rerun the upgrade
-command and it will resume from where it left off.
+fails or the upgrade is interrupted for any reason, rerun the upgrade
+command (the upgrade will resume from the point of interruption).
 
 
 #### Metadata Store Migration
 
-When upgrading from `2.5` to `2.6`, the system will run a `metadatastoremigration` job after a successful upgrade. This involves migrating the blob links for your images which is necessary for online garbage collection. With `2.6`, you can log in to the DTR web interface and navigate to **System > Job Logs** to check the status of the `metadatastoremigration` job. See [Audit Jobs via the Web Interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/) for more details.
+When upgrading from `2.5` to `2.6`, the system will run a `metadatastoremigration` job following a successful upgrade. This involves migrating the blob links for your images, which is necessary for online garbage collection. With `2.6`, you can log into the DTR web interface and navigate to **System > Job Logs** to check the status of the `metadatastoremigration` job. Refer to [Audit Jobs via the Web Interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/) for more details.
 
 ![](../images/migration-warning.png){: .with-border}
 
 Garbage collection is disabled while the migration is running. In the case of a failed `metadatastoremigration`, the system will retry twice.
 
 ![](../images/migration-error.png){: .with-border}
 
-If the three attempts fail, you will have to retrigger the `metadatastoremigration` job manually. To do so, send a `POST` request to the `/api/v0/jobs` endpoint:
+If the three attempts fail it will be necessary to manually retrigger the `metadatastoremigration` job. To do this, send a `POST` request to the `/api/v0/jobs` endpoint:
```

Comment:
"fail, it will be..."

Reply:
I don't really agree, but as adding the comma doesn't make it wrong...letcha have it. :-)